### DISCUSSION TITLE
Hola como se obtiene la insignia de Heart On Your Sleeve?

### DISCUSSION THREAD

**Discussion by Javi-CR** [community/community#132783] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

<img width="148" alt="68747470733a2f2f6769746875622e6769746875626173736574732e636f6d2f696d616765732f6d6f64756c65732f70726f66696c652f616368696576656d656e74732f68656172742d6f6e2d796f75722d736c656576652d64656661756c742e706e67" src="https://github.com/user-attachments/assets/85689594-27a8-45fe-9c67-94a979020772">

**Comment by Javi-CR** [comment]
❤️❤️

**Comment by davevad93** [comment]
Hi @Javi-CR ,

The achievements `Heart On Your Sleeve` and `Open Sourcerer` aren`t available yet.

**Reply by Javi-CR** [reply]
Que lastima, gracias por decirme 👌👌

**Comment by chemanishazad** [comment]
response #21962
